packages/bot/src/services/tradeEngine/trade/Balance.js

```diff
@@ -2,7 +2,7 @@ import { getFormattedText } from 'deriv-shared/utils/currency';
 import { info }            from '../utils/broadcast';
 import ScratchStore        from '../../../stores/scratch-store';
 
-let balanceStr = '';
+let balance_string = '';
```

Comment:
@aaron-binary is there any reason `balance_string` has been defined here?

Reply:
This is how our ancient trade engine keeps track of state. I've only updated the variable to conform to our coding styling guide here. When we do the trade engine refactor we can work on keeping state elsewhere.